src/harness/testcases/configs/test_WINNF_FT_S_PAT_1/default_fss.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Please reuse the existing testdata where possible. If you really need to add this file, it should not be in the configs/ folder since that folder should contain *only* configs and configs have a 1:1 mapping with test case execution.